src/game/shell.c

```diff
@@ -199,9 +199,21 @@ bool Shell_MakeScreenshot()
         break;
     }
 
-    char path[20];
+    char path[120];
     for (int i = 0; i < 10000; i++) {
-        sprintf(path, "screenshot%04d.%s", i, ext);
+#if defined(_WIN32)
+    #include <direct.h>
+        _mkdir("screenshots");
+#else
+    #include <sys/types.h>
+    #include <sys/stat.h>
+        mkdir("screenshots", 0664);
+#endif
+        char level_title[100];
+        sprintf(
+            level_title, "%s", g_GameFlow.levels[g_CurrentLevel].level_title);
```

Comment:
This has potential of creating illegal filenames.

https://stackoverflow.com/a/31976060
I suggest stripping all characters that are not A-Z or 0-9.

Reply:
Level title named `zs<> .,..,?!911 -lol-Caves` saved as `zs_911_lolCaves-0000.jpg` (haven't done timestamp yet). So I think I got the replace + strip done. Not as trivial to do in C as it is in Python haha.

Reply:
I knowwww I hate C for this. Thanks for persevering :heart:

Reply:
I'm going hard and naming it `Shell_ValidateLevelName`. Seems odd to only do it on screenshots. Right now, empty string levels and only bad char levels are accepted.